src/main/java/seedu/address/ui/InterviewPanel.java

```diff
@@ -0,0 +1,181 @@
+        attributesTab.setContent(attributeListPanel.getRoot());
+        questionListPanel = new QuestionListPanel(logic.getQuestionListView());
+        questionsTab.setContent(questionListPanel.getRoot());
+        metricListPanel = new MetricListPanel(logic.getMetricListView());
+        metricsTab.setContent(metricListPanel.getRoot());
+    }
+
+    /**
+     * Sets what is displayed in the listPanelStackPane based on the toggle.
+     *
+     * @param toggleView enum representing what should be displayed
+     */
+    public void handleToggle(ToggleView toggleView) {
+        // Clear the current interviewee
+        if (toggleView != ToggleView.TRANSCRIPT) {
+            logic.setCurrentInterviewee(null);
+        }
+
+
+        switch (toggleView) {
+
+        case INTERVIEWEE: // interviewee
+            intervieweePane.getChildren().clear();
+            intervieweePane.getChildren().add(intervieweeListPanel.getRoot());
+            intervieweePane.getChildren().add(sessionInformationCard.getRoot());
+            break;
+        case BEST_INTERVIEWEE:
+            intervieweePane.getChildren().clear();
+            BestIntervieweeListPanel bestNIntervieweesPanel =
+                    new BestIntervieweeListPanel(logic.getBestNIntervieweesView(), commandExecutor);
+            setAnchor(bestNIntervieweesPanel.getRoot(), 0.0, Double.NaN, 5.0, 5.0);
+            intervieweePane.getChildren().add(bestNIntervieweesPanel.getRoot());
+            intervieweePane.getChildren().add(sessionInformationCard.getRoot());
+            break;
+
+
+        case TRANSCRIPT: // transcript
+            transcriptPane.getChildren().clear();
+            Interviewee currentInterviewee = logic.getCurrentInterviewee();
+            DetailedIntervieweeCard detailedIntervieweeCard =
+                    new DetailedIntervieweeCard(currentInterviewee, commandExecutor);
+            RemarkListPanel remarkListPanel = new RemarkListPanel(currentInterviewee, logic.getQuestionListView());
+
+            transcriptPane.getChildren().add(remarkListPanel.getRoot());
+            setAnchor(remarkListPanel.getRoot(), 5.0, 0.0, 5.0, 5.0);
+
+            transcriptPane.getChildren().add(detailedIntervieweeCard.getRoot());
+            setAnchor(detailedIntervieweeCard.getRoot(), Double.NaN, 0.0, Double.NaN, 5.0);
+
+            break;
+
+        case ATTRIBUTE: // attribute
+            rubricsPane.getSelectionModel().select(attributesTab);
+            break;
+        case METRIC: // metrics
+            rubricsPane.getSelectionModel().select(metricsTab);
+            break;
+        case QUESTION: // questions
+            rubricsPane.getSelectionModel().select(questionsTab);
+            break;
+
+        default:
+            break;
+        }
+    }
+
+    /**
+     * Scrolls the Transcript to the given index.
+     *
+     * @param index the index to scroll to.
+     */
+    public void scrollTo(int index) {
+        remarkListPanel.scrollTo(index);
+    }
```

Comment:
The 'goto command' is supposed to scroll the desired remark but it does not work.

Reply:
<img width="1412" alt="Screenshot 2020-04-09 at 4 18 04 PM" src="https://user-images.githubusercontent.com/44172431/78873640-c30aff80-7a7d-11ea-8290-aee09bb7b71b.png">